test/integration/ipamd/ipamd_event_test.go

```diff
@@ -151,7 +151,8 @@ var _ = Describe("test aws-node pod event", func() {
 
 		It("unauthorized event must be raised on aws-node pod", func() {
 			By("waiting for event to be generated")
-			time.Sleep(5 * utils.PollIntervalLong)
+			// The event can take a long time to show up in client queries (I have seen up to 5 minutes)...
+			time.Sleep(5 * time.Minute)
```

Comment:
Would hard-coded `Sleep` 5 minutes for outliers be too expensive for our tests? Is that possible to make it polling for changes?

Reply:
Synced offline. We will have another PR on tests improvement including this in general.